PublicationsRouter/v2/api/API.md

```diff
@@ -199,18 +205,64 @@ Any of the notification endpoints listed below will return one of these response
 
 Note the last of these is different from the Validation endpoint.
 
-- On **authentication failure** (e.g. invalid api_key, incorrect user role) the system will respond with a 401 (Unauthorised) and no response body.
+- On **authentication failure** (e.g. invalid api_key, incorrect user role) the system will respond with a **401 (Unauthorised)** and no response body.
+
+```
+    HTTP 1.1  401 Unauthorized
+```
+
+- In the event of a **malformed HTTP request**, the system will respond with a **400 (Bad Request)** and the response body:
 
-- In the event of a **malformed HTTP request**, the system will respond with a 400 (Bad Request) and the response body:
 ```
     HTTP 1.1  400 Bad Request
     Content-Type: application/json
-    
+
     {
+        "status": "error",
         "error" : "human readable error message"
     }
 ```
+
+- In the event of a **Not Acceptable HTTP request** for a 'notification list', the system will respond with a **406 (Not Acceptable)** and the response body:
+
+```
+    HTTP 1.1  406 Not Acceptable
+    Content-Type: application/json
+
+    {
+        "successful": 0,
+        "success_ids": [],
+        "total": 2,
+        "fail_ids": [],
+        "last_error": "human readable error message"
+    }
+```
+
+- In the event of a **Partial Content HTTP request** for a 'notification list', the system will respond with a **206 (Partial Content)** and the response body:
+
+```
+    HTTP 1.1  206 Partial Content
+    Content-Type: application/json
+
+    {
+        "successful": 3
+        "success_ids": [1, 3, 5],
+        "total": 10,
+        "last_error": "human readable error message",
+        "fail_ids": [2, 4]
+    }
+```
+
+**NOTE**: It means that some notifications were processed before an error happened, so then the message will include the id of the latest notification processed. It might have happened that some notification were not processed and need to be resend. 
```

Comment:
Not clear to me what "It" refers to in this note. 

"latest notification processed" better as "last notification processed before error occured"

The last word should be "resent" not "resend".